include/caffe/layers/relu6_layer.hpp

```diff
@@ -0,0 +1,85 @@
+#ifndef CAFFE_RELU_LAYER_HPP_
```

Comment:
this macro collides the one from relu_layer.hpp this will lead to huge problems at some point